src/cryptonote_core/service_node_rules.h

```diff
@@ -97,6 +97,9 @@ namespace service_nodes {
   // blocks out of sync and sending something that it thinks is legit.
   constexpr uint64_t VOTE_OR_TX_VERIFY_HEIGHT_BUFFER    = 5;
 
+  // Minimum version of Storage Server required
+  constexpr int MIN_STORAGE_SERVER_VERSION[3] = {1, 0, 6};
```

Comment:
Can remove the array subscript since the compiler can deduce from the code.

Surely comment not necessary? The variable name is standard programming prose.

Reply:
Updated.